src/utils/github.ts

```diff
@@ -94,15 +94,21 @@ export class GithubApi {
   /**
    * Get content from the github repository
    */
-  async getContent(path: string): Promise<string> {
+  async getContent(path: string): Promise<string | null> {
     logger.info({ path }, 'GitHub API: Get Content');
-    const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
-    if (!this.isOk(response.status)) throw new Error('Failed to get aerial TileSet config.');
-    if ('content' in response.data) {
-      return Buffer.from(response.data.content, 'base64').toString();
-    } else {
-      throw new Error(`Unable to find the content from path ${path}.`);
+    try {
+      const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
+      if (this.isOk(response.status)) {
+        if ('content' in response.data) {
```

Comment:
Is it possible for `response.data.content` not to exist if the response is OK?

Reply:
This is not working as the content may not inthe data type.